web-console/4.0-designs/storage/storage.md

```diff
@@ -29,3 +29,52 @@
 ![form for adding and creating storage](img/create-storage-form.png)
 - Selecting `Create Storage` exposes fields for Storage Class, Name, Access Mode, Size, and the label selector checkbox.
 - Only access modes that are compatible should be available to select in this form.
+
+## Set Default Storage Class
+- Users with appropriate privileges can set a Default Storage Class.
+- THERE ARE TWO WAY USER CAN SET A DEFAULT STORAGE CLASS:
+
+#### OPTION-1: Kebab menu option:
+
+1. Login to OpenShift > Admin > Storage (menu) > Storage classes (sub-menu)
+2. User can see list of ‘Storage classes' [list view] & there will be a label that indicates `default` storage class
+
+3. Click on the kebab menu > select `Set as Default Storage Class`
+
+![Set a Default Storage Class using kebab menu](img/stroage-opt1-start-1.png)
+
+4. Pop-up opens with confirmation message.
+5. User clicks on 'Confirm' button to make the change
+
+![Popup menu with confirmation](img/stroage-opt1-popup-confirm-1.png)
+
+6. Successfully modified ‘Default Storage class” [list view]
+![Modified default storage class](img/stroage-opt1-done.png)
+NOTE: If user clicks on the kebab menu of the existing 'default storage class' then the `Set as Default Storage Class` will be disabled
+![Popup menu with confirmation](img/stroage-opt1-start-1-disable.png
+  )
+
+#### OPTION-2: Change the ‘default’ option from storage class detail page:
+
+1. Login to OpenShift > Admin > Storage (menu) > Storage classes (sub-menu)
+2. Click on a storage class name link and view details of the storage
+
+![Set a Default Storage Class using edit ](img/stroage-opt2-start.png)
```

Comment:
This looks like a repeat of the next image...is that intended?

Reply:
@lizsurette, 
I will create new .md as you mentioned (I will take Matt's help and do it). 
Regarding the repeated image... I need to change the image name.